server/app.js

```diff
@@ -347,8 +348,9 @@ module.exports = function createApp({
   app.use('/admin/', secureRoute(adminRouter()))
   app.use(
     '/admin/roUsers/',
-    secureRoute(userAdminRouter({ userAdminService, signInService, migrationService }), { auditKey: 'MIGRATION' })
```

Comment:
I've reverted this back to what it was before! didn't really make sense adding a MIGRATION specific key